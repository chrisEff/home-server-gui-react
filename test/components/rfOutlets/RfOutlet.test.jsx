'use strict'

const React = require('react')
const RfOutlet = require('../../../src/components/RfOutlets/RfOutlet')

const {shallow} = require('enzyme')

describe('RfOutlet', () => {
	it('should render correctly', () => {
		const wrapper = shallow(<RfOutlet outlet={{
			0: 12494204,
			1: 11940012,
			fauxmoPort: 11002,
			id: 1,
			name: 'Fernseher',
			protocol: 4,
			pulseLength: 355,
			state: 0,
		}} />)
		expect(wrapper).toMatchSnapshot()
	})
})
