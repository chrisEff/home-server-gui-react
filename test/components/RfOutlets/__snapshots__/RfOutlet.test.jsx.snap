// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RfOutlet should render correctly 1`] = `
<div
  className="device_1woep87-o_O-outlet_zf5bi4-o_O-off_1x9dv6u"
>
  <span>
    Fernseher
  </span>
  <div
    className="plug_15loxqg-o_O-plugOff_1pr3crw"
    onClick={[Function]}
  >
    ðŸ”Œ
  </div>
</div>
`;
